src/tests/ftest/io/parallel_io.py

```diff
@@ -192,3 +269,129 @@ def test_parallelio(self):
         # check dfuse is still running after attempting to access deleted
         # container.
             self.dfuse.check_running()
+
+    def test_multipool_parallelio(self):
+        """Jira ID: DAOS-3775.
```

Comment:
should this be DAOS-3776?

Reply:
As per conversation on Skype, will address it in upcoming ticket.